lib/xgboost/booster.rb

```diff
@@ -23,11 +20,7 @@ def save(path)
       FFI.XGBoosterSaveModel(handle_pointer, path)
     end
 
-    def free!
-      FFI.XGBoosterFree(handle_pointer)
-    end
-
-    def predict(input)
+    def predict(input, missing = 0)
```

Comment:
How about using kwargs here, and using NaN as default? `def predict(input, missing: Float::NAN)`